synapse/storage/databases/main/task_scheduler.py

```diff
@@ -193,7 +200,22 @@ async def get_scheduled_task(self, id: str) -> Optional[ScheduledTask]:
             desc="get_scheduled_task",
         )
 
-        return TaskSchedulerWorkerStore._convert_row_to_task(row) if row else None
+        return (
+            TaskSchedulerWorkerStore._convert_row_to_task(
+                (
+                    row["id"],
+                    row["action"],
+                    row["status"],
+                    row["timestamp"],
+                    row["resouce_id"],
+                    row["params"],
+                    row["result"],
+                    row["error"],
+                )
+            )
+            if row
+            else None
+        )
```

Comment:
I'm hoping to convert `simple_select_*` methods too, but that's a bigger job. So for now we have this ugliness.